# This msg file specifies the ee pose (position + orientation)
# for both arms (left and right arm)
# This msg file also indicates whether currently
# the hand of an arm is holding an object

geometry_msgs/Pose left_ee_pose
geometry_msgs/Pose right_ee_pose
bool isObjectInLeftHand
bool isObjectInRightHand
